src/plugins/foxPage/components/Total.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, Flex, Text } from '@chakra-ui/layout'
+import { SkeletonText } from '@chakra-ui/react'
+import { useTranslate } from 'react-polyglot'
+import { Amount } from 'components/Amount/Amount'
+import { AssetIcon } from 'components/AssetIcon'
+
+type TotalProps = {
+  icons: string[]
+  fiatAmount: string
+}
+
+export const Total = ({ icons, fiatAmount }: TotalProps) => {
+  const translate = useTranslate()
+
+  return (
+    <Box p={4}>
+      <Flex mb={6} flexDirection='row'>
+        {icons.map((icon, index) => (
+          <AssetIcon
+            key={icon}
+            src={icon}
+            boxSize='8'
+            // zIndex should be the decremental
+            zIndex={icons.length - (index + 1)}
+            ml={index > 0 ? -3.5 : 0}
```

Comment:
In general, floating point precision should be avoided (we have BN to circumvent that, which obviously can't be used here), especially given the fact that we're not in control of the logic of the used package. You can use a string instead